build.sbt

```diff
@@ -2,18 +2,14 @@ organization := "io.github.marad"
 name := "scalartemis"
 version := "1.0.0"
 
-scalaVersion := "2.11.6"
+scalaVersion := "2.13.0"
 
 resolvers += Resolver.jcenterRepo
 
-seq(bintraySettings:_*)
-seq(bintrayPublishSettings:_*)
-
 libraryDependencies ++= Seq(
-  "org.scalatest" %% "scalatest" % "2.2.1" % "test",
-  "org.scalacheck" %% "scalacheck" % "1.12.2" % "test",
-  "org.mockito" % "mockito-core" % "1.10.19" % "test"
+  "org.scalatest" %% "scalatest" % "3.0.8",
```

Comment:
Why did you remove the `% "test"`? Is this something about the new version?

Reply:
I actually mostly did it without thinking (I just pulled the latest version numbers for those libs), but after having researched what that actually does, these seem pointless. I think.

https://www.scala-sbt.org/1.x/docs/Library-Dependencies.html#Per-configuration+dependencies tells more of the story - they're basically dependencies that are configured only to apply on a `test` execution.

You don't have any tests in your project, meaning that you'd never run `test`... meaning that those dependencies are currently doing nothing. Unless you're doing some strange things externally, there's an argument to be made for removing them entirely.

Otherwise I can slap the `test` back on, but may as well remove what you're not using, right?

Reply:
I'm not sure why you think this project has no tests while there are bunch of them here: https://github.com/marad/scalartemis/tree/master/src/test/scala/io/github/scalartemis.

The other thing is... If you remove this `% test` it will add those dependencies to your resulting production JAR making it bigger with no reason, this libraries are only used in tests. So if you really thought there were no tests it would make more sense to remove them entirely.

But there are tests, so please add slap it back on ;)

Reply:
I'm sorry, I had a complete brainfart when I was browsing the codebase - I could've sworn I didn't see a `test` directory... 🤦‍♀ 

I'll add the `% test`s back in, my apologies.

Reply:
No problem. I haven't got time during the weekend to review the dependency issue yet. I'll try to look at it during the week somehowe.